CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+doc-548-submit-a-pull-request-section-to-contribution-guide
+```
+
+When `TYPE` can be:
+
+- **feat** - is a new feature
+- **doc** - documentation only changes
+- **cicd** - changes related to CI/CD system
+- **fix** - a bug fix
+- **refactor** - code change that neither fixes a bug nor adds a feature
+
+**All PRs must include a commit message with the changes description!**
+
+For the initial start, fork the project and use git clone command to download the repository to your computer. A standard procedure for working on an issue would be to:
+
+1. `git pull`, before creating a new branch, pull the changes from upstream. Your master needs to be up to date.
+
+```
+$ git pull
+```
+
+2. Create new branch from `main` like: `doc-548-submit-a-pull-request-section-to-contribution-guide`<br/>
+
+```
+$ git checkout -b [name_of_your_new_branch]
+```
+
+3. Work - commit - repeat ( be sure to be in your branch )
+
+4. Push changes to GitHub
+
+```
+$ git push origin [name_of_your_new_branch]
+```
+
+5. Submit your changes for review  
+   If you go to your repository on GitHub, you'll see a `Compare & pull request` button. Click on that button.
+6. Start a Pull Request  
+   Now submit the pull request and click on `Create pull request`.
+7. Get a code review approval/reject
+8. After approval, merge your PR
+9. GitHub will automatically delete the branch after the merge is done. (they can still be restored).
+## Installation
+
+To install a working development environment follow these instructions:
+
+1. Fork or clone the open-runtimes/open-runtimes repository.
+
+2. Install Composer dependencies using one of the following options:
+
+**Composer CLI**
+```bash
+composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+```
+
+**Docker (UNIX)**
+
+```bash
+docker run --rm --interactive --tty --volume "$(pwd)":/app composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+```
+
+**Docker (Windows)**
+
+```bash
+docker run --rm --interactive --tty --volume "%cd%":/app composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+```
+
+4. Build the runtimes you want to test using the docker build command:
+```
+cd ./runtimes/
+docker build -t {{Image Tag Name}} ./{{Folder of your runtime}}
+```
+
+3. [Follow our contribution guide to learn how you can add support for more runtimes.](https://github.com/appwrite/php-runtimes/blob/main/docs/add-runtime.md)
+
+## Introducing New Features
+
+We would 💖 for you to contribute to Open-runtimes, but we would also like to make sure Open-runtimes is as great as possible and loyal to its vision and mission statement 🙏.
+
+For us to find the right balance, please open an issue explaining your ideas before introducing a new pull request.
+
+This will allow the Open-runtimes community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
```

Comment:
```suggestion
This will allow the Open Runtimes community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
```